### PR TITLE
feat(start): start empty project

### PR DESCRIPTION
**Pull request by Ferkalek**
feat(start): start empty project

added empty structure to start project

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent implements OnInit{
+
+  constructor() {}
```
Comment (ArtemLunt) [inline src/app/app.component.ts:10]: Pls rid of redundant constructors, hooks, etc.

File: src/app/app.component.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
```
Comment (ArtemLunt) [inline src/app/app.component.ts:6]: Pls use OnPush all over.

Comment (ArtemLunt) [reply src/app/app.component.ts:6]: You can also specify `OnPush` as default strategy at `angular.json`.

File: src/app/app.module.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    AppComponent,
+    HomeComponent,
+    HeaderComponent
+  ],
+  imports: [
+    BrowserModule,
+    AppRoutingModule,
+    BrowserAnimationsModule,
+  ],
+  providers: [],
```
Comment (ArtemLunt) [inline src/app/app.module.ts:23]: Pls rid of redundant providers block.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
```
Comment (ArtemLunt) [inline package.json:13]: Pls provide also `ngxs` state manager and set it up.

File: src/app/home/home.component.spec.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (ArtemLunt) [inline src/app/home/home.component.spec.ts:1]: We don't need tests here, pls rid of spec files and create components/services/etc with `--skipTests`.

File: src/app/app.component.ts (hunk 1 above)
Comment (Ferkalek) [reply src/app/app.component.ts:10]: done

File: src/app/app.module.ts (hunk 3 above)
Comment (Ferkalek) [reply src/app/app.module.ts:23]: done

File: src/app/app.component.ts (hunk 2 above)
Comment (Ferkalek) [reply src/app/app.component.ts:6]: done

File: src/app/home/home.component.spec.ts (hunk 5 above)
Comment (Ferkalek) [reply src/app/home/home.component.spec.ts:1]: done

File: package.json (hunk 4 above)
Comment (Ferkalek) [reply package.json:13]: done

**Pull request merged**